source/code/general_types.md

```diff
@@ -51,15 +51,30 @@ _Example:_
 
 ## Languages and Transliterations
 
-Always use Unicode characters for any language or transliteration, along with one of the following four classes. Foreign book/article titles should only be styled with italics, not language specific styles such as `.lang` or `.translit`.
+We prefer using Unicode characters for foreign languages or transliterations, but in some circumstances, character entities are necessary to get things to render properly.
+
+For Greek and Hebrew, the **mywsb.css** classes `.lang-grc` and `.lang-hbo` will set the fonts to Gentium Plus and Ezra SIL, respectively.
 
 ```html
 <!-- Greek -->
 <span class="lang-grc">Ιεσουσ</span>
 
 <!-- Hebrew -->
 <span class="lang-hbo">‏יְשׁוּעָ</span>
+```
+
+Sometimes a combination of Hebrew and non-Hebrew characters has trouble rendering properly because Hebrew reads right-to-left. In these cases, it can be useful to use the `dir` attribute.
+Surrounding the entire phrase in an element with `dir="ltr"` and each series of Hebrew characters with an element having `dir="rtl"` can help.
+```html
+<!-- doesn't always render correctly -->
+<p>1 <span class="lang-hbo"dir="rtl">(ה)חד</span></p>
```

Comment:
You may want to put a space before `dir="rtl"` in `<span class="lang-hbo"dir="rtl">`